src/Digdir.Domain.Dialogporten.Janitor/ConsoleUser.cs

```diff
@@ -0,0 +1,9 @@
+using System.Security.Claims;
+using Digdir.Domain.Dialogporten.Application.Externals.Presentation;
+
+namespace Digdir.Domain.Dialogporten.Janitor;
+
+public class ConsoleUser : IUser
+{
+    public ClaimsPrincipal GetPrincipal() => throw new NotImplementedException();
```

Comment:
Hmm 🤔 På et tidspunkt må vi kanskje ta stilling til dette når Janitor bruker ressurser som er avhengig av IUser. Kanskje en liten melding i exceptionen om hvorfor dette er utelatt i denne omgang og fremgangmåte for å fikse dersom det blir et behov? 🤔

Reply:
La inn litt mer tekst.